packages/backend/src/config/app.ts

```diff
@@ -109,6 +109,9 @@ export const Config = {
     'AUTH_SERVER_SPEC',
     'https://raw.githubusercontent.com/interledger/open-payments/77462cd0872be8d0fa487a4b233defe2897a7ee4/auth-server-open-api-spec.yaml'
   ),
+  privateKey: parseOrProvisionKey(
+    envString('PRIVATE_KEY_PATH', '../../tmp/private.pem')
```

Comment:
```suggestion
    envString('PRIVATE_KEY_PATH', './tmp/private.pem')
```
If the `backend` is run from a working directory like `/root` the target directory isn't possible.